repo/packages/S/spark/12/marathon.json.mustache

```diff
@@ -3,6 +3,13 @@
     "cpus": {{service.cpus}},
     "mem": {{service.mem}},
     "cmd": "/sbin/init.sh",
+{{#security.authn.enabled}}
```

Comment:
@mgummelt calls this {{#security.mesos.authentication.enabled}} so it 1) doesn't get confused with spark's own UI's authentication, and 2) so we spell out authn